docs/src/man/convex_hulls.md

```diff
@@ -149,3 +163,45 @@ convex_hull([@SVector(rand(2)) for i in 1:3]) # warm-up
 v_static = [SVector{2, Float64}(vi) for vi in v]
 @time convex_hull(v_static)
 ```
+
+## Higher-dimensional convex hull
+
+One can compute the convex hull of points in higher dimensions using `convex_hull`.
+The appropriate algorithm is decided based on the dimensionality of the given
+points. The code looks the same as for dimension 2, and is not restricted to
+dimension 3; one can also use 4 or higher, in principle, although the computational
+complexity of `convex_hull` also increases.
```

Comment:
This sentence does not add any news, so I would drop it:
> The code looks the same as for dimension 2, and is not restricted to
dimension 3; one can also use 4 or higher, in principle, although the computational
complexity of `convex_hull` also increases.